File: Python/Program/PetCat.py
from Cat import Cat


class PetCat(Cat):
    def __init__(self, gender, favorite_toy, name):
        super().__init__(gender, "Carnivore", "Cat Ultramix", 5, 7, False)
        self.name = name
        self.favorite_toy = favorite_toy

    def display(self):
        print("PETCAT")
        self.display_info()

    def display_info(self):
        print(f"> Name: {self.name}")
        print(f"> Favorite Toy: {self.favorite_toy}")
        super().display_info()

File: Java/Program/PetCat.java

class PetCat extends Cat {
    private String name;
    private String favoriteToy;

    public PetCat(String gender, String favoriteToy, String name) {
        super(gender, "Carnivore", "Cat Ultramix", 5, 7, false);
        this.name = name;
        this.favoriteToy = favoriteToy;
    }

    public void display() {
        System.out.println("PETCAT");
        displayInfo();
    }

    public void displayInfo() {
        System.out.println("> Name: " + name);
        System.out.println("> Favorite Toy: " + favoriteToy);
        super.displayInfo();
    }
}